spec/controllers/shoutouts_controller_spec.rb

```diff
@@ -0,0 +1,207 @@
+require 'rails_helper'
```

Comment:
@VitaliiShevchenko in this file you still use removed fields: `recipients`, `digest`

Reply:
Thank you, deleted)